### ISSUE TITLE
Feature request: Add "warm_start_from" to tf.keras.estimator.model_to_estimator

### ISSUE THREAD

**Issue by bobbyAtSperry** [tensorflow/tensorflow#20057]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: Yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux
- **TensorFlow installed from (source or binary)**: Binary
- **TensorFlow version (use command below)**: 1.8
- **Python version**: 2.7
- **Bazel version (if compiling from source)**: N/A
- **GCC/Compiler version (if compiling from source)**: N/A
- **CUDA/cuDNN version**: N/A
- **GPU model and memory**: N/A
- **Exact command to reproduce**: N/A

### Describe the problem
I can create an estimator either direct or from a keras model using the initialisers:
```
tf.estimator.estimator(
    model_fn,
    model_dir=None,
    config=None,
    params=None,
    warm_start_from=None
)
```
```
tf.keras.estimator.model_to_estimator(
    keras_model=None,
    keras_model_path=None,
    custom_objects=None,
    model_dir=None,
    config=None
)
```
However with the keras approach there is no option to warm start the model from a previously saved checkpoint. I am requesting a feature that makes it possible to do this with a call such as:
```
tf.keras.estimator.model_to_estimator(
    keras_model=None,
    keras_model_path=None,
    custom_objects=None,
    model_dir=None,
    config=None,
    warm_start_from=None
)
```

Without this I cannot see how to start an estimator in a pre-trained state.

I have also tried to do this by just calling an untrained estimator to do .predict which causes it to attempt to load from a checkpoint. When I provide the checkpoint from previous estimator training I get an error as not all data seems to be available. This is described in more detail at stackoverflow here https://stackoverflow.com/questions/50855256/keras-estimator-model-to-estimator-cannot-warm-start-or-load-previous-checkpoi

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by bhack** [comment]
A duplicate of https://github.com/tensorflow/tensorflow/issues/19295?
Another related problem is at https://github.com/tensorflow/tensorflow/issues/18696

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by bhack** [comment]
It is a duplicate.

**Comment by shivaniag** [comment]
Closing this since its a duplicate.
